src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java

```diff
@@ -367,7 +368,9 @@ public void tick() {
 				//Replaced with for loop because in crystal hollows with events the line it's on can shift.
 				for (String line : lines) {
 					if (line.contains("⏣")) {
-						location = Utils.cleanColour(line).replaceAll("[^A-Za-z0-9() ]", "").trim();
+						String l = Utils.cleanColour(line).replaceAll("[^A-Za-z0-9() ]", "").trim();
+						if(!l.equals(location)) { new ScoreboardLocationChangeListener(location, l); }
```

Comment:
im not sure why we use a constructor here.

Reply:
Idk I'm bad at stuff, you can change it if you want

Reply:
I dont want to, i just wanted to needlessly complain